### ISSUE TITLE
Identify Ubuntu version automatically?

### ISSUE THREAD

**Issue by evandam** [erlef/setup-beam#279]
#### Is your feature request related to a problem?

Not related to a problem, but it would be nice if we could avoid having to set the `ImageOS` env var and the action could detect it for us (at least some of the time).

#### Describe the feature you'd like

For Ubuntu-based runners, it would be great if the action could just read from `/etc/os-release` things like `ID=ubuntu` and `VERSION_ID=24.04` for example, rather than users setting `ImageOS=ubuntu24` explicitly.

#### Describe alternatives you've considered

Setting `ImageOS` works, just a thought for a small improvement 😄 

#### Additional context

Hopefully it makes sense and would be helpful!

**Comment by paulo-ferraz-oliveira** [comment]
Makes sense to me. The action could then throw only if you haven't been able to deduce it from the OS.

**Comment by paulo-ferraz-oliveira** [comment]
Support for self-hosted runners is very limited at this moment, and this seems to not have gained traction. Closing as "Won't do now". Feel free to re-open for further discussion (especially if you can provide code with it 👍).
